src/selection-recommendation/recommend.py

```diff
@@ -207,23 +208,134 @@ def process_schedule(all_times, all_sections):
+    |___________|___________|___________|___________|___________|___________|
+    | 5:00pm    | -         | -         | -         | -         | -         |
+    |           |           |           |           |           |           |
+    |___________|___________|___________|___________|___________|___________|
+    | 6:00pm    | CSC148    | CSC165    | CSC165    | CSC148    | -         |
+    |           | Lec 5101  | Lec 5101  | Lec 5101  | Lec 5101  |           |
+    |___________|___________|___________|___________|___________|___________|
+    | 7:00pm    | CSC148    | CSC165    | CSC165    | CSC148    | -         |
+    |           | Lec 5101  | Lec 5101  | Lec 5101  | Lec 5101  |           |
+    |___________|___________|___________|___________|___________|___________|
+    | 8:00pm    | -         | CSC165    | -         | CSC148    | -         |
+    |           |           | Lec 5101  |           | Lec 5101  |           |
+    |___________|___________|___________|___________|___________|___________|
+    | 9:00pm    | -         | -         | -         | -         | -         |
+    |           |           |           |           |           |           |
+    |___________|___________|___________|___________|___________|___________|
+
+    """
+    print(" " + "_"*71)
+    print("|           " + "| MONDAY".ljust(12) + "| TUESDAY".ljust(12) + "| WEDNESDAY".ljust(12) + "| THURSDAY".ljust(12) + "| FRIDAY".ljust(12) + "|")
+    print("|___________"*6 + "|")
+
+    for i in range(28):
+        if i % 2 == 0:
+            if i//2 <= 3:
+                print("|", (str(i//2 + 8) + ":00am").ljust(10), end="")
+            elif i//2 == 4:
+                print("|", "12:00pm".ljust(10), end="")
+            else:
+                print("|", (str(i//2 - 4) + ":00pm").ljust(10), end="")
+        else:
+            print("|", "          ", end="")
+
+        for j in range(5):
+            if not schedule[j][i//2]:
+                if j != 4:
+                    if i % 2 == 0:
+                        print("| -".ljust(12), end="")
+                    else:
+                        print("|           ", end="")
+                elif i % 2 == 0:
+                    print("| -".ljust(12) + "|")
+                else:
+                    print("|           |")
+                    print("|___________"*6 + "|")
+            else:
+                if i % 2 == 0:
+                    if j != 4:
+                        print(("| " + schedule[j][i//2][0]).ljust(12), end="")
+                    else:
+                        print(("| " + schedule[j][i//2][0]).ljust(12) + "|")
+                else:
+                    if j != 4:
+                        print(("| " + schedule[j][i//2][1]).ljust(12), end="")
+                    else:
+                        print(("| " + schedule[j][i//2][1]).ljust(12) + "|")
+                        print("|___________"*6 + "|")
+
+
+def get_best_schedule(campus, *args):
```

Comment:
Can you give out all the valid schedules and let user select their favourite? It's possible that the only best schedule given is unavailable due to no spots